server/src/core/user/types.ts

```diff
@@ -46,6 +58,14 @@ export interface UserService {
    * @param args 
    */
   create(args: UserCreateArgs): Promise<User>;
+  /**
+   * nominates an exisiting user as a beneficiary
+   * only if they're not a donor. If user does not
+   * exist, however, a user account is created 
+   * with the role 'beneficiary'
+   * @param args 
+   */
+  nominate(args: UserNominateArgs): Promise<User>;
```

Comment:
This should be more specifically named (.e.g `addBenficiary` or `inviteBeneficiary` or `nominateBeneficiary`)